cmd/minishift/cmd/root.go

```diff
@@ -174,22 +187,30 @@ func init() {
 	hasEnabledExperimental = enableExperimental
 
 	RootCmd.PersistentFlags().Bool(showLibmachineLogs, false, "Show logs from libmachine.")
+	RootCmd.PersistentFlags().String(profile, "", "Profile name")
```

Comment:
what's about the default? What would happen if we literally make the default 'minishift' here. Would that not simplify some code?